integration/internal/api/container/container.go

```diff
@@ -0,0 +1,128 @@
+package container
+
+import (
+	"context"
+	"io"
+	"os"
+	"time"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	networktypes "github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
+)
+
+// CreateVMemLabel creates a container with the supplied parameters
+func CreateVMemLabel(client client.APIClient, v []string, memoryMB int, labels map[string]string, img string, cmd []string) (string, error) {
```

Comment:
I think all of these can be replaced with helpers that return `container.Config()` instead.

Reply:
This would break the predictable homogeneity and make invocations of what used to be `docker run` much more verbose.